### PR TITLE
add .idea and .vscode into gitignore

### PR DESCRIPTION
**Pull request by Hikariks**
add .idea and .vscode into gitignore

### CODE REVIEW

**Review by kroscedu — APPROVED** [review]

**Review by fuelatank — APPROVED** [review]

**Review by GaryZhangstu — APPROVED** [review]

File: src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java:1]: "The error exception" is grammatically wrong. "Unknwon Error" should be "Unknown Error".

File: src/main/java/cn/msa/msa_museum_server/exception/BusinessException.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/exception/BusinessException.java:1]: For the code style, the empty line in the constructor should probably be removed, and a space should be added between `)` and `{`.

File: src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java:1]: "The error exception" is grammatically wrong. "Unknwon Error" should be "Unknown Error".

File: src/main/java/cn/msa/msa_museum_server/exception/ExceptionEnum.java
Comment (fuelatank) [inline src/main/java/cn/msa/msa_museum_server/exception/ExceptionEnum.java:1]: I think the error code of `MISSING_PARAMETERS` should probably be at the `1000` level, for example `1001`

File: src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java
Comment (fuelatank) [reply src/main/java/cn/msa/msa_museum_server/exception/ErrorHandler.java:1]: Why 1949?

**Pull request closed without merging**